File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int value;
    do
    {
        value = get_int("Height: ");
    }
    while (value < 1 || value > 8);

    for (int i = 1; i <= value; ++i, printf("\n"))
    {
        for (int space = 1; space <= value - i; ++space)
        {
            printf(" ");
        }

        for (int hashtag = 1; hashtag <= i; ++hashtag)
        {
            printf("#");
        }

        for (int space = 1; space <= 2; ++space)
        {
            printf(" ");
        }

        for (int hashtag = 1; hashtag <= i; ++hashtag)
        {
            printf("#");
        }
    }
}

File: pset6/mario.py
from cs50 import get_int

height = get_int("Height: ")
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(1, height + 1):
    j = height - i
    for a in range(j):
        print(" ", end="")
    for a in range(i):
        print("#", end="")
    print("  ", end="")
    for a in range(i):
        print("#", end="")
    print()
